src/main/java/bio/terra/cloudfunctions/common/MediaTypeWrapper.java

```diff
@@ -0,0 +1,51 @@
+package bio.terra.cloudfunctions.common;
+
+import com.google.common.net.MediaType;
+
+public class MediaTypeWrapper {
+  // Workaround for Mime types not found in Guava MediaType
+  // See issue https://github.com/google/guava/issues/3946
+  public static final MediaType APPLICATION_GZIP = MediaType.create(MediaType.GZIP.type(), "gzip");
+  public static final MediaType APPLICATION_JSON =
```

Comment:
Let's make these fields private since they're not used outside of the class.

Reply:
Updated scope to private.